plugins/catalog-backend/package.json

```diff
@@ -32,11 +32,13 @@
   "dependencies": {
     "@backstage/backend-common": "^0.10.1",
     "@backstage/catalog-client": "^0.5.3",
-    "@backstage/plugin-catalog-common": "^0.1.0",
     "@backstage/catalog-model": "^0.9.8",
     "@backstage/config": "^0.1.11",
     "@backstage/errors": "^0.1.5",
     "@backstage/integration": "^0.7.0",
+    "@backstage/plugin-auth-backend": "^0.6.1",
```

Comment:
Realized this dependency is a bit unfortunate, perhaps we could just copy over `IdentityClient.getBearerToken` for now? Feeling we need this as a common util as it ends up all over the place

Reply:
Sounds good, copied.